tests/runtime/index.ts

```diff
@@ -39,12 +42,31 @@ export class Mappings {
     return new Mappings(abi, host, instance)
   }
 
-  private handler(name: string, ...args: unknown[]): void {
-    ;(this.instance.exports[name] as (...a: unknown[]) => void)(...args)
+  private handler(name: string, event: Event): void {
+    const eventPtr = this.abi.writeEvent(event)
+    ;(this.instance.exports[name] as (...a: unknown[]) => void)(eventPtr)
```

Comment:
This line is some advanced ts/js. Why is the function type not `(a: Pointer) => void`?

Reply:
No, just forgot to change it from before.